diff --git a/node_modules/@silveredgold/beta-censor-client/client.js b/node_modules/@silveredgold/beta-censor-client/client.js
index b857104..b42b8ac 100644
--- a/node_modules/@silveredgold/beta-censor-client/client.js
+++ b/node_modules/@silveredgold/beta-censor-client/client.js
@@ -4,7 +4,6 @@ import { SimpleEventDispatcher } from "@silveredgold/ste-simple-events";
 import { HttpTransportType, HubConnectionState } from "@microsoft/signalr";
 import { HubConnectionBuilder } from "@microsoft/signalr";
 import { log } from "missionlog";
-import fetch from "cross-fetch";
 import { dbg, dbgLog, dbgTime } from "./util";
 export class BetaCensorClient {
     /**
